google/cloud/bigtable/client_options.cc

```diff
@@ -24,6 +24,11 @@
 #define BIGTABLE_CLIENT_DEFAULT_CHANNELS_PER_CPU 2
 #endif  // BIGTABLE_CLIENT_DEFAULT_CHANNELS_PER_CPU
 
+// Overwrite default message limit size of 4MiB to 256MiB
```

Comment:
Is this comment necessary? The old value may have been 4MiB, but it doesn't add any additional info if someone is performing a diff. Plus, the comment may become outdated.